src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -227,7 +357,13 @@ void CascadiaSettings::_WriteSettings(const std::string_view content)
 {
     auto pathToSettingsFile{ CascadiaSettings::GetSettingsPath() };
 
-    auto hOut = CreateFileW(pathToSettingsFile.c_str(), GENERIC_WRITE, FILE_SHARE_READ | FILE_SHARE_WRITE, NULL, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
+    auto hOut = CreateFileW(pathToSettingsFile.c_str(),

```

Comment:
Uh.... isn't `auto hOut` then just a bare `HANDLE`?

Shouldn't this be a `wil::unique_handle` so it gets closed properly if an error is thrown by `WriteFile`?